pkg/users/users.js

```diff
@@ -174,12 +174,17 @@ async function getLogins() {
     const lastlog = await cockpit.spawn([LastLogPath], { environ: ["LC_ALL=C"] });
 
     let currentLogins = [];
+    const logind = cockpit.dbus("org.freedesktop.login1");
```

Comment:
Don't want to push for this to happen in this PR but, I'd love to also get watching happening via logind:

Replacing:
```
        let handleUtmp;
        if (utmppath !== null) {
            // Watch `/var/run/utmp` or `/var/lib/wtmpdb/wtmp.db` to register when user logs in or out
            handleUtmp = cockpit.file(utmppath, { superuser: "try", binary: true });
            handleUtmp.watch(() => debouncedGetLogins(), { read: false });
        }
```

Reply:
So we have a users global logind handle with signals for that.